c++/cavl.hpp

```diff
@@ -32,13 +34,17 @@
 /// be costly in terms of execution time.
 #ifndef CAVL_ASSERT
 #    if defined(CAVL_NO_ASSERT) && CAVL_NO_ASSERT
-#        define CAVL_ASSERT(x) (void) 0
+#                                       // NOLINTNEXTLINE(cppcoreguidelines-macro-usage) function-like macro
+#        define CAVL_ASSERT(x) (void) 0 /* NOSONAR cpp:S960 */
 #    else
-#        include <cassert>  // NOLINTNEXTLINE function-like macro
-#        define CAVL_ASSERT(x) assert(x)
+#        include <cassert>
+#                                        // NOLINTNEXTLINE(cppcoreguidelines-macro-usage) function-like macro
```

Comment:
Please remove the leading `#` here and on line 37

![image](https://github.com/pavel-kirienko/cavl/assets/3298404/59ab612d-ba33-420e-8b41-5da621a50727)